server/athenian/api/openapi/openapi.yaml

```diff
@@ -7841,3 +7890,375 @@ components:
       - scale
       - ticks
       type: object
+    CommitPushers:
+      description: Check runs must be triggered by commits pushed by these people.
+        When it is impossible to determine who pushed, e.g. in legacy API based checks,
+        they are committers.
+      example:
+      - github.com/vmarkovtsev
+      - github.com/se7entyse7en
+      items:
+        $ref: '#/components/schemas/DeveloperID'
+      type: array
+    FilterCodeChecksRequest:
+      additionalProperties: false
+      description: Request body of `/filter/code_checks`.
+      example:
+        in:
+        - github.com/athenianco/athenian-webapp
+        - github.com/athenianco/athenian-api
+        triggered_by:
+        - github.com/vmarkovtsev
+        - github.com/se7entyse7en
+        date_from: 2020-05-01
+        date_to: 2021-05-01
+        timezone: 120
+        account: 1
+      properties:
+        in:
+          $ref: '#/components/schemas/RepositorySet'
+        triggered_by:
+          $ref: '#/components/schemas/CommitPushers'
+        date_from:
+          description: Date from when to start measuring the metrics.
+          format: date
+          type: string
+        date_to:
+          description: Date up to which to measure the metrics.
+          format: date
+          type: string
+        timezone:
+          $ref: '#/components/schemas/TimeZone'
+        account:
+          description: Session account ID.
+          type: integer
+      required:
+      - account
+      - date_from
+      - date_to
+      - in
+      type: object
+    FilteredCodeCheckRuns:
+      additionalProperties: false
+      description: |-
+        Response from `/filter/code_checks`, found code check runs ordered by repository
+        name and then by title. Note: we always consider the completed run executions only.
+      example:
+        items:
+        - title: Unit tests
+          repository: github.com/src-d/athenian-api
+          last_execution: 2021-05-18
+          size_groups:
+          - 3
+          - 5
+          - 8
+          total_stats:
+            count: 100
+            successes: 50
+            mean_execution_time: 200s
```

Comment:
We'll need quantiles applied to this one.

Reply:
Can I hardcode 0.95 maybe? How about the timelines?

Reply:
Hardcoding 0.95 would work for a pretty long time I believe. Yes please do.
For timelines it should work as the regular timelines in the product ie. apply quantization overall then expand the values through the timeline. And not apply quantization separately to each data point eg. each day

Reply:
I realized that hardcoding 0.95 is almost equivalent to exposing `quantiles`. Added `quantiles`.